src/main/webapp/index.html

```diff
@@ -47,12 +47,14 @@
 <link rel="stylesheet" href="https://unpkg.com/nouislider@10.0.0/distribute/nouislider.min.css" />
 
 <!-- Custom -->
+<link rel="stylesheet" href="css/style.css" />
```

Comment:
Just curious why the separate style.css and script.js files?  Are there more specific names that could be used?

Reply:
I put some of the code common to multiple pages in the style.css and script.js files. Would a name like "shared.css" be fine?

Reply:
"shared.css" or "common.css" seems fine to me.